fe/fe-core/src/test/java/com/starrocks/sql/analyzer/PrivilegeCheckerV2Test.java

```diff
@@ -1578,6 +1578,16 @@ public void testSetStmt() throws Exception {
         String sql = "SET PASSWORD FOR 'jack'@'192.%' = PASSWORD('123456');";
         String expectError = "Access denied; you need (at least one of) the GRANT privilege(s) for this operation";
         verifyNODEAndGRANT(sql, expectError);
+
+        ctxToTestUser();
+        // user 'test' not has GRANT/NODE privilege
+        sql = "set password = PASSWORD('123456')";
+        StatementBase statement = UtFrameUtils.parseStmtWithNewParser(sql, starRocksAssert.getCtx());
+        PrivilegeCheckerV2.check(statement, starRocksAssert.getCtx());
```

Comment:
if user 'test' doesn't have grant privilege on system, it will throw exception here?

Reply:
no, set your own password, no permission is required, here you can pass check